CMPUT_301_Project/app/src/main/java/com/example/cmput_301_project/SignUpFragment.java

```diff
@@ -0,0 +1,97 @@
+package com.example.cmput_301_project;
+
+import static androidx.core.content.ContextCompat.getSystemService;
+
+import android.content.Context;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+
+import android.view.KeyEvent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.FrameLayout;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link SignUpFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class SignUpFragment extends Fragment  implements View.OnClickListener {
+
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
```

Comment:
I'm not sure what "param1" and "param2" are. Maybe username/password? Can we rename these to something more descriptive?

Reply:
params created when default fragment was created and serve no purpose atm. Removed in newest commit.